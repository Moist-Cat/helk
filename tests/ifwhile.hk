prints("if-while");
let d = 10;
while (d) {
    let d = d - 1;
    if (min(max(d - 5, 0), 1)) {
        prints("foo");
    }
    else {
        0;
    };
};
