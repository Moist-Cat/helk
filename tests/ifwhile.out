if-while
foo
foo
foo
foo
